reviewrot/basereview.py

```diff
@@ -1,8 +1,41 @@
 from collections import OrderedDict
+import datetime
+from dateutil.relativedelta import relativedelta
 
 
 class BaseService(object):
 
+    def format_reviews(self, username, title, url, comments, created_at,
+                       state_, value, duration):
+        """
+        find the relative time difference between now and
+        review request filed to retrieve relative information
+        """
+        rel_diff = relativedelta(datetime.datetime.now(),
+                                 created_at)
+        """
+        find the absolute time difference between now and
+        review request filed to retrieve absolute information
+        """
+        abs_diff = datetime.datetime.now() - created_at
+        if (state_ is not None and value is not None and
```

Comment:
I think I pretty much had that way in the previous version. If I remove the check part from here, then nothing remains in 'format_reviews' method. There is already a method that checks if the review should be included. I am just going to remove abs_diff and rel_diff is calculations inside the existing method that checks if the review should be included, so considering this suggestion, basically we are not removing redundancy from github, gitlab, pagure etc. files but just moving the calculations for more readability.